shell/common/shell.cc

```diff
@@ -986,8 +990,11 @@ void Shell::OnPlatformViewSetViewportMetrics(int64_t view_id,
       });
 
   {
+    // TODO(dkwingsmt): Use the correct view ID when this method supports
+    // multi-view.
+    int64_t view_id = kFlutterImplicitViewId;
```

Comment:
This method already accepts a `view_id` argument. Should we remove this local variable?